addons/mail/static/src/messaging/component/chatter_topbar/chatter_topbar.scss

```diff
@@ -54,11 +62,16 @@
 .o_ChatterTopbar .o-active {
     color: $o-brand-odoo;
     background-color: lighten(gray('300'), 7%);
-    border-color: gray('300');
+    border-right-color: gray('300');
+    border-left-color: gray('300');
     border-bottom-color: transparent;
     border-top-color: transparent;
 }
 
+.o_ChatterTopbar_button:hover {
+    background-color: gray('300');
```

Comment:
The topbar buttons are actually rounded, so this is not pixel perfect.
Similar issue is prevent with "active" button.

I think the radius should always be removed from the bottom corners?

Reply:
Maybe from top corners too actually, are they really wanted?

Reply:
Actually it's not a rounding it's due to the fact that border top and bottom are transparent. The best would be to make them "none". 'm on it.

Reply:
The issue is still there and I'm sure it's because of border radius, you can zoom in a lot if you don't see it with normal view. The problem is fixed for me if I just remove all border radius.

Reply:
Okay I think it's only on community as I only looked en enterprise, I'll look on friday